### ISSUE TITLE
Segmentation fault 

### ISSUE THREAD

**Issue by Menogus** [APN-Pucky/tyrant_optimize#12]
i compile it:
root@astraG:/home/adming/tyrant/tyrant_optimize-3.0.2# make VERSION=v3.0.2
```
make -f make/Makefile.linux all
make[1]: вход в каталог «/home/adming/tyrant/tyrant_optimize-3.0.2»
mkdir -p obj
touch obj/.stamp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/sim_test.o -c sim_test.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/tyrant.o -c tyrant.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/read.o -c read.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/cards.o -c cards.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/xml.o -c xml.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/sim.o -c sim.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/deck.o -c deck.cpp
g++ -Wall -Werror -std=gnu++11 -Ofast -DNDEBUG -DNQUEST -DTYRANT_OPTIMIZER_VERSION='"v3.0.2"' -o obj/tyrant_optimize.o -c tyrant_optimize.cpp
g++ -o tuo obj/sim_test.o obj/tyrant.o obj/read.o obj/cards.o obj/xml.o obj/sim.o obj/deck.o obj/tyrant_optimize.o -lboost_system -lboost_thread -lboost_filesystem -lboost_regex -lboost_timer -lpthread
```

and when i run tuo
_./tuo "menog_u" "ShellGauntlet" pvp ordered -v -o -s climbex 10 200 fund 0 -L 4 10 dom- -e "SuperHeroism"_

he has error: 

**Segmentation fault**

**Comment by Menogus** [comment]
when i post it on kongregate forum, admins have banned me!! :(((

(https://www.kongregate.com/forums/2468-general/topics/920187-tuo-v2-63-2?page=14)

**Comment by APN-Pucky** [comment]
I fixed a crash-bug with v3.0.4. Maybe it works now. Also your comment on kong-forums is visible again.

**Comment by Menogus** [comment]
thank you!:))
all work fine now
